packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -66,6 +62,11 @@
       }
     }
 
+    .#{$prefix}--card-link__title,
+    .#{$prefix}--card-link__content {
+      width: 90%;
```

Comment:
I wonder if there is a way we could create a or some mixins to help with some of this? Either just a single mixin to help with these couple of values, or maybe our own type mixins that wrap in this kind of stuff? Just thinking out loud at this point I guess.